instrumentation/aws-lambda/aws-lambda-core-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambdacore/v1_0/internal/ApiGatewayProxyRequest.java

```diff
@@ -38,17 +40,22 @@ private static boolean xrayPropagationFieldsOnly(Collection<String> fields) {
             fields.iterator().next());
   }
 
-  public static ApiGatewayProxyRequest forStream(InputStream source) throws IOException {
-
+  public static ApiGatewayProxyRequest forStream(InputStream source) {
     if (noHttpPropagationNeeded()) {
       return new NoopRequest(source);
     }
 
     if (source.markSupported()) {
       return new MarkableApiGatewayProxyRequest(source);
     }
-    // fallback
-    return new CopiedApiGatewayProxyRequest(source);
+    // It is known that the Lambda runtime passes ByteArrayInputStream's to functions, so gracefully
```

Comment:
I believe this is reasonable. Notably, if we were actually hitting the fallback codepath, it would often be failing with ClassNotFoundException since commons-io is not referenced at all by lambda libraries and would only have to be available by luck